Algorithm/QCAlgorithm.cs

```diff
@@ -1827,6 +1827,12 @@ private T AddSecurity<T>(SecurityType securityType, string ticker, Resolution re
                 symbol = QuantConnect.Symbol.Create(ticker, securityType, market);
             }
 
+            Security existingSecurity;
+            if (Securities.TryGetValue(symbol, out existingSecurity))
+            {
+                throw new Exception($"The security '{symbol.Value}' has already been added.");
```

Comment:
I think we should use the `ticker` here, since this is what the user directly specified, maybe something like this that regurgitates the user's input?
``` csharp
throw new Exception($"{ticker} {securityType} has already been added.")
```